dspace/config/spring/api/scripts.xml

```diff
@@ -50,4 +50,9 @@
         <property name="description" value="Perform the media filtering to extract full text from documents and to create thumbnails"/>
         <property name="dspaceRunnableClass" value="org.dspace.app.mediafilter.MediaFilterScript"/>
     </bean>
+
+    <bean id="item-database-status" class="org.dspace.app.itemdbstatus.ItemDatabaseStatusCliScriptConfiguration">
```

Comment:
While I understand why you call this script `item-database-status`, I have to admit I find the name confusing.  That name makes it sound like the script is updating the status of the Item _in the database_.  

But, in reality it isn't changing/touching the database at all. Instead, it's cleaning up your Solr index by "resyncing" it with the database.

I realize this might be a large change to make, but I honestly wonder if this script should be called something like `item-solr-cleanup` or `solr-database-resync` or something like that.  Unfortunately though, that name change would touch a lot of this PR (including package names).  But, I do feel the current name is misleading because it doesn't mention Solr at all (and the whole purpose of this script is to cleanup your solr index)

Reply:
solr-database-resync indeed seems like a more appropriate name, so I renamed the script, classes, tests and packages to match this.